SixTrack/sixtrack.s

```diff
@@ -61777,11 +61777,21 @@ c$$$            endif
          mat = 6
       elseif (c_material.eq.'C2') then
          mat = 7
+      elseif (c_material.eq.'MoGR') then
+         mat = 8
+      elseif (c_material.eq.'CuCD') then
+         mat = 9
+      elseif (c_material.eq.'Mo') then
+         mat = 10
+      elseif (c_material.eq.'Glid') then
+         mat = 11
+      elseif (c_material.eq.'Iner') then
+         mat = 12
 !02/2008 TW added vacuum and black absorber (was missing) 
       elseif (c_material.eq.'VA') then
```

Comment:
Actually, there is no material n. 13 at the moment. 
The empty slot I left was just to accommodate another material without changing once again the 'nmat' index throughout the code.